src/webgl/shaders/cubeVertex.vert

```diff
@@ -0,0 +1,13 @@
+// Adapted from:  https://learnopengl.com/PBR/IBL/Diffuse-irradiance
+layout (location = 0) in vec3 aPos;
```

Comment:
This might also need to be changed into `IN vec3 aPos` in order to also work in WebGL 1 environments